resources/js/beatmap-discussions/renderers.tsx

```diff
@@ -21,14 +19,15 @@ export function linkRenderer(astProps: ReactMarkdownProps & React.DetailedHTMLPr
 }
 
 export function paragraphRenderer(astProps: ReactMarkdownProps & React.DetailedHTMLProps<React.HTMLAttributes<HTMLParagraphElement>, HTMLParagraphElement>) {
-  return <div className='beatmapset-discussion-message'>{astProps.children.map(timestampDecorator)}</div>;
+  // TODO: remove need for wrapping div (should just return <>);
+  return <div>{astProps.children.map(timestampDecorator)}</div>;
```

Comment:
more like should be a `p` instead of `div` for non-review thing (mainly for the margin)